go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/ghodss/yaml v1.0.0
 	github.com/gogo/protobuf v1.3.2
-	github.com/golang/protobuf v1.5.2
+	github.com/golang/protobuf v1.4.3
```

Comment:
Is it desirable to change to a lower version? Same as below.

Reply:
Thanks! I didn't notice that change, reverted.